weblab/entities/views.py

```diff
@@ -721,3 +744,96 @@ def get_context_data(self, **kwargs):
             kwargs['formset'] = self.get_formset()
         kwargs['type'] = self.object.entity_type
         return super().get_context_data(**kwargs)
+
+
+class EntityDiffView(View):
+    def _get_unix_diff(self, file1, file2):
```

Comment:
Would it be worth creating a new (low priority) issue to use https://docs.python.org/3/library/difflib.html for this instead do you think?

Reply:
I wanted to use that, but it doesn't seem to provide the same style of diff as the `diff` command (i.e. to match the output given by WL1), unless I missed something.